solutions/src/main/kotlin/de/thermondo/solutions/basics/Basics06Solution.kt

```diff
@@ -0,0 +1,14 @@
+package de.thermondo.solutions.basics
+
+/**
+ * Create a class named Basics6Solution
+ *
+ * Define member variables:
+ * name: String
+ * age: Int
+ * and return the data of the two people while writing unit tests.
```

Comment:
```suggestion
 * @param name: String
 * @param age: Int
```

Let's use the @param keywords to define the parameters that we will be passing to the constructor.